src/app/pages/PerpetualPage/components/AccountBalanceForm/index.tsx

```diff
@@ -58,7 +62,7 @@ export const AccountBalanceForm: React.FC<AccountBalanceFormProps> = ({
         valueLabel: (
           <AssetValue
             value={available}
-            assetString="BTC"
+            assetString={pair.baseAsset}
```

Comment:
BTC was right here, as this relates to the collateral not the base.
```suggestion
            assetString="BTC"
```